spec/features/merchants/items/index_spec.rb

```diff
@@ -6,39 +6,99 @@
+
+      let!(:item_6) { create(:item, merchant_id: merchant_1.id, status: 1)}
+      let!(:invoice_11) { create(:invoice, customer_id: customer_1.id, status: 1)}
+      let!(:invoice_12) { create(:invoice, customer_id: customer_1.id, status: 1)}
+      let!(:invoice_item_11) { create(:invoice_item, invoice: invoice_11, item: item_6, unit_price: 1200000, quantity: 1) }
+      let!(:invoice_item_12) { create(:invoice_item, invoice: invoice_12, item: item_6, unit_price: 1200000, quantity: 1) }
+
+      let!(:transaction_1) { create(:transaction, invoice: invoice_1, result: 0) }
+      let!(:transaction_2) { create(:transaction, invoice: invoice_2, result: 0) }
+      let!(:transaction_3) { create(:transaction, invoice: invoice_3, result: 0) }
+      let!(:transaction_4) { create(:transaction, invoice: invoice_4, result: 0) }
+      let!(:transaction_5) { create(:transaction, invoice: invoice_5, result: 0) }
+      let!(:transaction_6) { create(:transaction, invoice: invoice_6, result: 0) }
+      let!(:transaction_7) { create(:transaction, invoice: invoice_7, result: 0) }
+      let!(:transaction_8) { create(:transaction, invoice: invoice_8, result: 0) }
+      let!(:transaction_9) { create(:transaction, invoice: invoice_9, result: 0) }
+      let!(:transaction_10) { create(:transaction, invoice: invoice_10, result: 0) }
+      let!(:transaction_11) { create(:transaction, invoice: invoice_11, result: 0) }
+      let!(:transaction_12) { create(:transaction, invoice: invoice_12, result: 0) }
+
+      let!(:item_6) { create(:item, merchant_id: merchant_2.id, status: 0)}
+      let!(:item_7) { create(:item, merchant_id: merchant_1.id, status: 0)}
 
       # User Story 6 - Merchant Items Index Page
 
       it "displays a list of names of all that merchants items" do
         visit "/merchants/#{merchant_1.id}/items"
-        expect(page).to have_content(item_1.name)
-        expect(page).to have_content(item_2.name)
-        expect(page).to have_content(item_3.name)
-        expect(page).to_not have_content(item_4.name)
+
+        within ".enabled-items" do
+          expect(page).to have_content(item_1.name)
+          expect(page).to have_content(item_2.name)
+          expect(page).to have_content(item_3.name)
+          expect(page).to have_content(item_4.name)
+          expect(page).to have_content(item_5.name)
+
+          expect(page).to_not have_content(item_6.name)
+        end
 
         visit "/merchants/#{merchant_2.id}/items"
-        expect(page).to have_content(item_4.name)
-        expect(page).to_not have_content(item_1.name)
-        expect(page).to_not have_content(item_2.name)
-        expect(page).to_not have_content(item_3.name)
+        within ".disabled-items" do
+          expect(page).to have_content(item_6.name)
+          expect(page).to_not have_content(item_1.name)
+          expect(page).to_not have_content(item_2.name)
+          expect(page).to_not have_content(item_3.name)
```

Comment:
Nothing major, but you tested that ".enabled-items" has items 1-5 but in ".disabled-items" you only tested it doesn't have items 1-3.